### PR TITLE
[sobjectizer] Update to v.5.7.3

### PR DESCRIPTION
**Pull request by eao197**
SObjectizer and its companion project so5extra are updated to the latest versions.

### CODE REVIEW

File: ports/so5extra/portfile.cmake [hunk 1]
```diff
@@ -1,8 +1,8 @@
 vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
     REPO stiffstream/so5extra
-    REF 20f4c83ecde1509fbaf337dcf40f2f49dcf2690d # v.1.4.1.1
-    SHA512 a3df042b60afc4c57361b5b3c21f4b7c077f1b0ab7a4d33fda14cc915f10b22a42ef0acbb1c7c8b356ce31ee84f24391164120642faf96235549204c83b40294
+    REF 713ed4876135dfaa9b744b567f7c300eae09800d # v.1.5.0
+    SHA512 51b1e9521288c4cfbbf29aa9719b9da3ee0073e38af6fc275a5ec0a22b4bededf293b136aac0cf99a435b4411ccf0687556418fc25285f501a6f426f3a623c34
 )
 
 vcpkg_configure_cmake(
```
Comment (JonLiu1993) [inline ports/so5extra/portfile.cmake:8]: @eao197 ,Thanks for your pr, Could you modernize these cmake functions?

Comment (eao197) [reply ports/so5extra/portfile.cmake]: Yes, fixed.

Comment (JonLiu1993) [reply ports/so5extra/portfile.cmake]: I will help you modernize cmake functions

Comment (eao197) [reply ports/so5extra/portfile.cmake]: I hope I fixed them at e3c2400a77f3693d9344a010a8c1c22dfef72396.

Comment (JonLiu1993) [reply ports/so5extra/portfile.cmake]: Where your modification is correct but not complete, I am almost done

Comment (eao197) [reply ports/so5extra/portfile.cmake]: Thank!

**Review by JonLiu1993 — APPROVED** [review]

**Review by JonLiu1993 — APPROVED** [review]

**Comment by strega-nil-ms** [comment]
Thanks!

**Pull request merged**